packages/launchpad/cypress/e2e/integration/choose-a-browser.spec.ts

```diff
@@ -0,0 +1,262 @@
+import type { FoundBrowser } from '@packages/types/src'
+
+describe('Choose a Browser Page', () => {
+  // Walks through setup pages to get to browser selection
+  const stepThroughConfigPages = () => {
+    cy.get('h1').should('contain', 'Configuration Files')
+    cy.contains('Continue').click()
+    cy.get('h1').should('contain', 'Initializing Config...')
+    cy.contains('Next Step').click()
+  }
+
+  // Mocks server's browser detection with known list
+  const setupMockBrowsers = function () {
+    cy.withCtx(async (ctx, o) => {
+      const mockBrowsers = [{
```

Comment:
I ended up writing a utility to mock out these browsers: https://github.com/cypress-io/cypress/pull/19375#discussion_r770792840

In the interest of keeping this PR more stable, I won't duplicate that here. But this mocking will get updated sooner or later depending on what we want to merge first.